src/components/custom-field-input-single-choice/custom-field-input-single-choice.test.jsx

```diff
@@ -99,6 +99,21 @@ describe('src/components/custom-field-input-single-choice/custom-field-input-sin
       userEvent.click(screen.getByLabelText('Test label'));
       expect(screen.queryByText('No options available.')).not.toBeInTheDocument();
     });
+
+    it('does not break if the choices are changed', () => {
+      const { rerender } = render(<CustomFieldInputSingleChoice {...requiredProps} choices={[]} />);
+      userEvent.click(screen.getByLabelText('Test label'));
+      expect(screen.getByText('No options available.')).toBeInTheDocument();
+
+      rerender(<CustomFieldInputSingleChoice {...requiredProps} choices={[{ id: '1', label: 'yo' }]} />);
```

Comment:
Do we need to assert that the new choices are available?

Reply:
That's a good idea, as it is it only tests for an error being thrown. Your suggestion will make the intent more specific!